src/release.md

```diff
@@ -3,13 +3,57 @@ __Thursday 28th March 2024__
 
 ### What's New
 
-- "Subscribe to changes" in apps 
-- Documentation across the platform
-- New option to export a datastore without including the data within tables
+- **"Subscribe to changes" in apps**
+
+_Subscribe to changes_ is a new feature in Apps that replaces polling. Previously, if you wanted data to update on your app page you would set a polling interval so that the data was refreshed at a frequent interval. This had limitations such as only being able to set a minimum interval of 1 second and also it could result in your App doing a lot more work than it had to.
+This new feature means that your App page will now be notified when data has been updated or workflow results have been returned. This allows your App to update automatically as soon as the data has updated and means that your app users see the data update in real time and also your App will be more efficient as it is only making updates when it needs to. 
+This will work with the following data sources:
+1. Any datastore variable, including tables
+2. Results from workflows that have been run via an App
+
+> You will not need to make any changes to start using this feature, App components which previously had configured polling will now automatically Subscribe to changes.
+
+Look out for this feature when adding a datasource to an App component!
+
+:img{src="/src/assets/subscribe_to_changes.png", alt="Subscribe to changes", width="100px"}
+
+- **Documentation across the platform**
+
+The platform now comes with built in documentation so you can learn about features and functionality without ever having to leave the application. This includes documentation within the platform itself, usually available by selecting the little `( i )` information icon next to entities. It also includes documentation for actions, app components, app actions and connectors.
+
+This also applies to when you are creating new entities within the Toca Development Kit, you are now required to add documentation before submitting your component for review. This ensures that all entities will continue to be documented.
+
+The platform documentation is also hosted in [GitHub](https://github.com/tocalabs/toca-docs) and this is a public repository so you are welcome to browse the documentation from there and even contribute if you wish to.
+
+Keep your eyes peeled the info icon that signifies documentation!
+
+:img{src="/src/assets/docs_example.png", alt="Documentation in platform", width="100px"}
```

Comment:
@chrismilway 100px width might be too small - no idea what was a decent figure to use. I don't want the images to dwarf the text, especially now since we can zoom into images. Open to suggestion/feedback

Reply:
The drawer this doc appears in has a max-width of 640px, so if the image looks OK at that size then I'd drop the width. Otherwise whatever size you want 🤷